torch/onnx/_internal/exporter/_capture_strategies.py

```diff
@@ -198,22 +204,31 @@ class TorchExportNonStrictStrategy(CaptureStrategy):
     def _capture(
         self, model, args, kwargs, dynamic_shapes
     ) -> torch.export.ExportedProgram:
-        try:
-            return torch.export.export(
-                model, args, kwargs=kwargs, dynamic_shapes=dynamic_shapes, strict=False
-            )
-        except torch._dynamo.exc.UserError as exc:
-            # Refine the dynamic shapes based on the suggested fixes.
+        with (
+            _patch_dynamo_unsupported_functions(),
```

Comment:
@justinchuby I added this context manager as well, but can you confirm that you missed it for NonStrictStrategy? I thnk this happened because at the time we thought strict=False was not important.

Reply:
I looked at _patch_dynamo_unsupported_function. Seems like it’s just handling jit isinstance. Since non strict is tracing without dynamo, the isinstance call will just be executed without problem.

So I don’t think it is needed for non strict.

Reply:
Ok I will remove it then.

Reply:
removed